uk_bin_collection/uk_bin_collection/councils/EastHampshireDC.py

```
@@ -0,0 +1,203 @@
+                try:
+                    gpdf_path = self._get_pdf(gpdf_url)
+                    gdates = self._parse_pdf(gpdf_path)
+                    today = datetime.now().replace(hour=0, minute=0, second=0, microsecond=0)
+                    for dt in gdates:
+                        if dt >= today:
+                            data["bins"].append({
+                                "type": "Garden Waste",
+                                "collectionDate": dt.strftime(date_format),
+                            })
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)